docs/extendedusage.rst

```diff
@@ -198,6 +200,23 @@ Traits
     'The index of the first track to return'
 
 
+RAML1.0 Types
+~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    >>> api.types
+    {'Person': ObjectType(name='Person', properties={'name': Property(type='string')})}
+    >>> person = api.types['Person']
+    >>> person.type
```

Comment:
well this is actually the case, its inheriting from the base type "object"
I need to think a little bit more on how the type inheritance should be described